css-fonts-4/Overview.bs

```diff
@@ -2908,6 +2908,71 @@ Initial: normal
 
 This descriptor defines initial settings that apply when the font defined by an @font-face rule is rendered. It does not affect font selection. Values are identical to those defined for the 'font-language-override!!property' property defined below except that the value inherit is omitted. When multiple font feature descriptors, properties, or variations are used, the cumulative effect on text rendering is detailed in the section [[#font-feature-variation-resolution]] below.
 
+<h3 id="font-metrics-override-desc">
+Default font metrics overriding:
+the 'ascent-override', 'descent-override' and 'line-gap-override' descriptors</h3>
+
+<pre class='descdef'>
+Name: ascent-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+<pre class='descdef'>
+Name: descent-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+<pre class='descdef'>
+Name: line-gap-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+The 'ascent-override', 'descent-override' and 'line-gap-override' descriptors define the
+<a spec="CSS-INLINE-3">ascent metric</a>, <a spec="CSS-INLINE-3">descent metric</a> and
+<a spec="CSS-INLINE-3">line gap metric</a> of the font.
+
+When the descriptor value is 'normal', the corresponding metric value is obtained from the
+font file directly.
+
+Note: User agents may choose different pieces of data from the font file as the metric value,
+leading to different layouts. For example, Chrome on Windows uses sTypoAscender, sTypoDescender
+and sTypoLineGap from the font's OS/2 table, while on Mac, it uses those in the HHEA table if
+available.
+
+When the descriptor value is a percentage, the corresponding metric value is resolved as the
+given percentage multiplied by the used font size. Negative values are illegal.
```

Comment:
s/illegal/ignored as invalid/ maybe?

Reply:
It's important to mention what "illegal" or "invalid" mean. We probably want them to be parse-time errors. So if you have
```
ascent-override: 60%;
ascent-override: -70%;
```
the value that gets used is the 60%.

Reply:
```suggestion
given percentage multiplied by the used font size. Negative values are invalid at parse time.
```